README.md

```diff
@@ -1,134 +1,38 @@
-and grant Travis permissions over your repository.
-
-### Initialize NodeJS/NPM
-1. If you haven't already, install NodeJS and NPM on your computer.
-
-2. Initialize your repository for NPM by running `npm init`. This will generate a `package.json` file in your repository.
-
-### Choose a Web Framework and Install It
-Within the NodeJS runtime, there a many frameworks for creating
-server applications. For the purposes of this assignment, there are two options you should consider:
-
-* <b>Option 1: Express</b><br>
-The [Express Framework](https://expressjs.com/) is a general-purpose web
-development framework with widespread adoption.
-
-* <b>Option 2: Loopback</b><br>
-The [Loopback Framework](https://loopback.io/) is a purpose-built REST API framework, with a smaller user base.
-
-As a deliverable of sprint 0, you will be asked to compare these frameworks. You
-should familiarize yourself with both projects in order to justify your decision.
-Once the decision is made, install it in your groups repository.
-
-### QA/Helpful Tools
-1. You should install a linter for your repository, to help manage code style.
-We highly recommend [eslint](https://eslint.org/docs/user-guide/getting-started)
-or [tslint](https://palantir.github.io/tslint/) if you decide to use typescript.
-
-To go a little further, we can use [eslint-watch](https://www.npmjs.com/package/eslint-watch) to automatically lint
-while we're programming:
-
-   > Success Condition:
-   > ```
-   > $ npm run lint (## which is calling esw -w src test)
-   > > ✓ Clean (10:12:27 AM)
-   > ```
-
-2. Because NodeJS projects have many dependencies, it is massively beneficial to
-use a tool to detect dependency updates and alert you as to potential
-vulnerabilities. Normally, we would recommend using a SaaS tool like
-[GreenKeeper](https://greenkeeper.io/) or [requires.io](https://requires.io/).
-However, if you're repository is private, these tools require payment. If
-that's the case, you should install [npm-check](https://www.npmjs.com/package/npm-check).
-
-   > Success Condition:
-   > ```
-   > $ npm-check
-   > > ❤️  Your modules look amazing. Keep up the great work. ❤️
-   > ```
-
-3. You should setup a test framework within your application to help with later
-test-driven development. We'd suggest [Jest](https://jestjs.io/) (especially for React
-development). Another option is to go with the [Mocha](https://mochajs.org/)
-framework along with [Chai](http://chaijs.com/). Once installed, write a
-single test, which doesn't actually test anything (besides that your tests run).
-
-    > Success Condition:
-    > ```
+# Dronuts
+### Ryan Barrett, Derek Yan, Judy Zhang
+This is Group 3's project for CMU's Software Engineering for Startups class (17-356). This readme will describe how to run and test the Dronuts application.
+
+## How to run the application for...
+### Development
+
+To run the application in development mode, run the following code from the root directory:
```

Comment:
I believe `npm start` should be run in `/Dronuts` instead of the root directory

Reply:
Should probably mention running `npm install` too

Reply:
Will make the changes